### DISCUSSION TITLE
Moving public/bundles to a S3 CDN (Ingress issues)?

### DISCUSSION THREAD

**Discussion by jantomicky** [pimcore/pimcore#16251] [Q&A / Support] [answered]
Is there any way of loading the contents of (as hard copies I assume) public/bundles from an S3 CDN instead of a local storage?
We're having issues with Kubernetes Ingress where it refuses to load the bundle files and instead tries to search for an application route (which obviously does not exist for static files).

**Comment by dpfaffenbauer** [comment, marked as answer]
Depending on your Kubernetes Setup, you have to copy the public/bundles file into your container. In our case, we have an nginx container and an fpm container, we copy the static files into both containers.

**Comment by jantomicky** [comment]
Thank you!
